gateway/1.5/modules/ROOT/pages/policies-pdk-develop-custom-policies.adoc

```diff
@@ -2,4 +2,84 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+After installing all the xref:policies-pdk-prerequisites.adoc[], you can begin developing custom policies.
+
+The custom policy lifecycle consists of the following docs that explain all steps of developing a custom policy that occur in the Policy Development Kit (PDK). After you xref:policies-pdk-create-project.adoc[Create a New Project], you can complete the steps out of order, for example, when debugging a policy you may need to return to the configure policy feature or define a schema definition steps.
```

Comment:
Instead of "you can complete the steps out of order", you might say something like "you might iteratively complete some of the steps. For example, ....

Another way to show this is to use numbered lists for things that must be done in order and use bulleted lists for things that don't. 

In general, I think it's understood in technical procedures that steps might be done in a different order, since that is often the case. You are just showing them one suggested way to do it.